### PR TITLE
Add pre-release changelog notes

### PR DESCRIPTION
**Pull request by Zmoki**

### CODE REVIEW

**Comment by Zmoki** [comment]
[Effects Tab: выпустить новую версию](https://trello.com/c/0jmpc0h2/267-effects-tab-%D0%B2%D1%8B%D0%BF%D1%83%D1%81%D1%82%D0%B8%D1%82%D1%8C-%D0%BD%D0%BE%D0%B2%D1%83%D1%8E-%D0%B2%D0%B5%D1%80%D1%81%D0%B8%D1%8E)

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+* Support for new configuration options of the widget,
```
Comment (dayton1987) [inline CHANGELOG.md]: Support for new widget configuration options: `<option-1>`, `<option-2>`. Learn more `[here](<reference>)`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+  [`previewProxy`](https://uploadcare.com/docs/uploads/widget/config/#option-preview-proxy)
+  and [`previewUrlCallback`](https://uploadcare.com/docs/uploads/widget/config/#option-preview-url-callback),
+  read more about it [here](https://uploadcare.com/docs/uploads/widget/secure_urls/).
+* Added French (`fr`) locale, thanks to [@gpenverne](https://github.com/gpenverne).
```
Comment (dayton1987) [inline CHANGELOG.md]: French (`fr`) locale thanks to `[@handle](<reference>)`.

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+
+### Changed
+
+* Updated README:
+  * updated links,
+  * added Feedback section,
+  * removed Contributors section.
```
Comment (dayton1987) [inline CHANGELOG.md]: * References.
* Added section, Feedback.
* Removed section, Contributors.

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed
+
+* Autorotate for image preview, [#21][pr-21].
+
+[pr-21]: https://github.com/uploadcare/uploadcare-widget-tab-effects/pull/21
```
Comment (dayton1987) [inline CHANGELOG.md]: `[github-pr-21]`? — since our main changelog will be composed using this one as a base.

**Review by dayton1987 — CHANGES_REQUESTED** [review]

**Pull request merged**